hyp3lib/rtc2color.py

```diff
@@ -43,26 +68,30 @@ def rtc2color(fullpolFile, crosspolFile, threshold, geotiff, cleanup=False,
   print('Estimated Total RAM usage = {} GB'.format(size*16))
 
   # Read full-pol image
-  print('Reading full-pol image (%s)' % fullpolFile)
+  print('Reading full-pol image (%s)' % out_tif)
   data = fullpol.GetRasterBand(1).ReadAsArray()
   cp = (data[:rows, :cols]).astype(np.float16)
   data = None
   cp[np.isnan(cp)] = 0
+  # FIXME: do we get negative powers from RTC?
   cp[cp < 0] = 0
```

Comment:
We're guarding against negative values here, but power shouldn't ever be negative. Do we get negative values ever out of GAMMA RTC? Should we let this fail if we do?

Reply:
Yes it does.